persistent/Database/Persist/Sql/Migration.hs

```diff
@@ -27,6 +27,7 @@ import Control.Monad.Trans.Control (liftBaseOp_)
 import Database.Persist.Sql.Types
 import Database.Persist.Sql.Raw
 import Database.Persist.Types
+import Database.Persist.Sql.Orphan.PersistStore()
```

Comment:
This import is necessary for the `BackendCompatible SqlBackend SqlBackend` instance.